src/translations/en.json

```diff
@@ -4491,7 +4491,7 @@
             "latitude": "Latitude",
             "longitude": "Longitude",
             "passive": "Passive",
-            "passive_note": "Passive zones are hidden in the frontend and are not used as location for device trackers. This is useful if you just want to use it for automations.",
+            "passive_note": "Passive zones are hidden in the frontend and not shown as a location of people or device trackers. This is useful if you just want to use them for automations.",
```

Comment:
may be "persons" instead of "people"?

Reply:
Hmm, in German we use "Personen", but in English it's "People" in settings.
In automations it's only "someone or something" in the descriptions.